README.md

```diff
@@ -125,7 +125,10 @@ you are interested in hearing about it.
 
 ## How to use the **@coqbot** instance ##
 
-To use the bot without deploying your own instance, follow these steps:
+### As a GitHub App
```

Comment:
Please add a warning that installing as a GitHub App is still in the experimental stage and that users may frequently receive requests to expand permissions.